generators/server/templates/app/config/grafana/provisioning/datasources/datasource.yml

```
@@ -1,51 +1,51 @@
+apiVersion: 1
+
+# list of datasources that should be deleted from the database
+deleteDatasources:
+  - name: Prometheus
+    orgId: 1
+
+# list of datasources to insert/update depending
+# whats available in the database
+datasources:
+  # <string, required> name of the datasource. Required
+- name: Prometheus
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)